frontend/ui/widget/touchmenu.lua

```diff
@@ -26,6 +26,10 @@ local util = require("ffi/util")
 local _ = require("gettext")
 local Screen = Device.screen
 local getMenuText = require("util").getMenuText
+local FocusManager = require("ui/widget/focusmanager")
```

Comment:
See https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires

FocusManager, Event and UnderlineContainer should go up above.

Input can stay here, one line above Screen.